app/services/concerns/case_worker_claim_param_readable.rb

```diff
@@ -0,0 +1,82 @@
+module CaseWorkerClaimParamReadable
```

Comment:
💭 **[Just FYI, maybe for a future PR]** Related with https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2352/files#r184657292, if we were to build, let's say for the sake of this example a `UpdatableCaseWorkerClaim` object in `CaseWorkerClaimUpdater` with the params we received, all these methods would be methods of `UpdatableCaseWorkerClaim` instance, so would be the validations, with no need for sharing concerns across classes, since this _claim_ instance would respond to all of it:

```ruby
class UpdatableCaseWorkerClaim
  include ActiveModel::Validations

  validates_with Claims::ClaimStateTransitionValidator

  def initialize(claim, params)
    @claim = claim
    @params = params 
  end

  def current_user
    @current_user ||= params[:current_user]
  end
  
  ...
end

# in  CaseWorkerClaimUpdater
record = UpdatableCaseWorkerClaim.new(claim, params)
record.valid?
```

Just an idea.